gitversion

```diff
@@ -0,0 +1,41 @@
+description = "GitVersion calculates your version based on your Git history."
+binaries = ["gitversion"]
+test = "gitversion -version"
+
+linux {
+  arch = amd64
+  source = "https://github.com/GitTools/GitVersion/releases/download/${version}/gitversion-linux-x64-${version}.tar.gz"
+  strip = 1
+}
+
+linux {
+  arch = arm64
+  source = "https://github.com/GitTools/GitVersion/releases/download/${version}/gitversion-linux-arm64-${version}.tar.gz"
+}
+
+darwin {
+  arch = amd64
+  source = "https://github.com/GitTools/GitVersion/releases/download/${version}/gitversion-osx-x64-${version}.tar.gz"
+}
+
+darwin {
```

Comment:
This will work fine, so no real need to change it, but the newer format for this is:

```
platform darwin arm64 {
  // ...
}
```

Reply:
Made the changes and ran `hermit test gitversion --trace`
![image](https://github.com/cashapp/hermit-packages/assets/10780503/ab90e3a4-ab0a-4c5f-acef-b02b7f01a6b7)

Seems like I can't merge this though.